coral/src/app/features/topics/components/search/SearchTopics.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Flexbox,
+  Icon,
+  InputBase,
+  PrimaryButton,
+} from "@aivenio/design-system";
+import { FormEvent, useState } from "react";
+import searchItem from "@aivenio/design-system/dist/module/icons/search";
+
+type SearchTopicsProps = {
+  searchTerm: string;
+  search: (searchTerm: string) => void;
+};
+
+function SearchTopics(props: SearchTopicsProps) {
+  const { search, searchTerm } = props;
+  const [currentSearchTerm, setCurrentSearchTerm] =
+    useState<string>(searchTerm);
+
+  function hasInput() {
+    return currentSearchTerm.trim().length > 0;
+  }
+
+  function onSubmit(event: FormEvent) {
+    event.preventDefault();
+    if (!hasInput()) {
+      return;
+    }
+    const trimmed = currentSearchTerm.trim();
+    setCurrentSearchTerm(trimmed);
+    search(trimmed);
+  }
+
+  return (
+    <form role={"search"} onSubmit={onSubmit} aria-label={"Topics"}>
```

Comment:
I've a note on my desktop to ask about that!

`Form` does not take any property for setting a specific role or aria-label, and the landmark is important in that case.

Reply:
Should I add that to our implementation and we can show that to console as example?

Reply:
I think we should aim to use our existing form abstraction. Consistency is king and otherwise we would never fix the issues in the root. E.g. if the accessibility is not good enough, we should aim to fix it in either the `<Form />` or `<Input>` or in DS.

Reply:
So, we would give the feedback for `Form` to console & update our `Form` and then use it when it's updated in console's implementation?

The search in DS is very different from our implementation, so we can provide the feedback, but it would be a big change that also would have to have an ok from the design team. We can provide the feedback there and see how it goes? Until then I would use our implementation for the search, since it provides better accessibility.

Reply:
Ok, lets go with the current implementation. I think this could be considered as technical debt, and it is totally fine to take it consciously. We just should either document / communicate when we diverge from the default path or take the shortcuts.

Reply:
OK! I'll make myself a note to get back at the teams next week (DS and console) and add a housekeeping issue here so we have it in our board.